test/e2e/register_orchestrator_test.go

```diff
@@ -30,8 +30,13 @@ func TestRegisterOrchestrator(t *testing.T) {
 
 func startOrchestrator(t *testing.T, geth *gethContainer) *livepeer {
```

Comment:
It would be interesting in general to move all the partial flows to `e2e.go` / dedicated directory. IMO, this will help reuse these instead of heading to write test-specific ones. Our test-writing-flow should involve looking for and using appropriate partial-flow from the ones already there, **then** if none is suitable, try to add params to close/similar existing ones or; if it's not possible write a new one.